tornettools/generate_defaults.py

```diff
@@ -49,8 +56,7 @@
 
 TGENRC_SERVER_FILENAME="tgen-server.tgenrc.graphml"
 TGENRC_PERFCLIENT_FILENAME="tgen-perf.tgenrc.graphml"
-TGENRC_MARKOVCLIENT_DIRNAME="tgen-markov"
-TGENRC_MARKOVCLIENT_FILENAME_FMT="{}.tgenrc.graphml"
+TGENRC_MARKOVCLIENT_FILENAME="tgenrc.graphml"
 TGENRC_FLOWMODEL_FILENAME_FMT="flowmodel.{}usec.graphml"
```

Comment:
Nit: should we use this instead to be consistent with the stream and packet model names?

```
TGENRC_FLOWMODEL_FILENAME_FMT="tgen.tor-flowmodel-{}usec.graphml"
```

Reply:
Done.